x-pack/plugins/search_notebooks/public/components/notebooks_button.tsx

```diff
@@ -30,7 +30,7 @@ export const SearchNotebooksButton = ({
   if (activeView) {
     return (
       <EuiButton
-        color="success"
+        color="accentSecondary"
```

Comment:
@mdefazio it looks like EUI won't be supporting `accentSecondary` as a button. Do you mind sharing the screenshot so we can help you decide what the right way to go is here? cc @ryankeairns

Reply:
Pasted in following comment